qr-code/src/main/java/com/adyen/checkout/qrcode/QrCodeUIEvent.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 10/1/2023.
+ */
+
+package com.adyen.checkout.qrcode
+
+sealed class QrCodeUIEvent {
```

Comment:
Could this be internal? If not let's annotate it with library only visibility

Reply:
It can't be internal, will annotate it with library scope.